.github/workflows/ci.yml

```diff
@@ -0,0 +1,26 @@
+# This workflow will build a Java project with Gradle
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-gradle
+
+name: CI
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: serializing workflow runs
+      uses: softprops/turnstyle@v1
+      with:
+        same-branch-only: false
+      env:
+        GITHUB_TOKEN: ${{ secrets.GH_TOKEN }}
+    - name: Set up JDK 1.8
+      uses: actions/setup-java@v1
+      with:
+        java-version: 8
+    - name: Build with Maven
+      run: mvn verify
+      env:
+        CTP_PROJECT_CONFIG: ${{ secrets.CTP_PROJECT_CONFIG }}
```

Comment:
I think these Environment variables are wrong.
The correct once`s are:
- IT_CLIENT_ID  
- IT_CLIENT_SECRET  
- IT_PROJECT_KEY  (payment-to-order-processor-7)

Reply:
https://github.com/commercetools/commercetools-payment-to-order-processor/pull/85/commits/75921e89c1b9141e3bbfad10bf6c21c9f204c5fd